curriculum/challenges/english/03-front-end-libraries/react-and-redux/manage-state-locally-first.english.md

```diff
@@ -107,11 +109,13 @@ class DisplayMessages extends React.Component {
     });
   }
   submitMessage() {
-    const currentMessage = this.state.input;
-    this.setState({
+    this.setState((state) => ({
+      currentMessage: state.input
+    }));
+    this.setState((state) => ({
```

Comment:
There should not be two calls to `this.setState` within the same function/method. These should be combined.